src/EnergyPlus/DXCoils.hh

```diff
@@ -1595,7 +1596,7 @@ namespace DXCoils {
 
 	//     NOTICE
 
-	//     Copyright � 1996-2014 The Board of Trustees of the University of Illinois
+	//     Copyright � 1996-2014 The Board of Trustees of the University of Illinois
```

Comment:
Copyright symbol got botched...why does it say 2014, anyway :wink:?  You don't have to change this, I have a branch around here somewhere to update the year to 2015, and I'll update it to remove the symbol entirely.